CHANGELOG.md

```diff
@@ -69,6 +69,7 @@
 - Added support for `Session.stored_procedure_profiler`.
 
 #### Improvements
+- Improved the following new capability for function `snowflake.snowpark.functions.array_remove` it is now possible to use it with a string.
```

Comment:
nit: this should go to the unreleased note

Reply:
Now is move to new new features.